android/build.gradle

```diff
@@ -10,8 +10,15 @@ android {
     }
 }
 
+def safeExtGet (prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 dependencies {
     implementation 'com.facebook.react:react-native:[0.40,)'
-    implementation 'com.urbanairship.android:urbanairship-fcm:9.5.6'
-    implementation 'com.google.firebase:firebase-core:16.0.5'
+    implementation ('com.urbanairship.android:urbanairship-fcm:9.5.6') {
+      exclude group: 'com.google.firebase', module: 'firebase-messaging'
+    }
+    implementation "com.google.firebase:firebase-messaging':${safeExtGet('firebaseMessagingVersion', '17.3.4')}"
```

Comment:
The error is `> Could not find com.google.firebase:firebase-messaging':17.3.4.`
looks like we have a rogue `'` here `messaging':$`

Reply:
Thanks! (No gradle syntax highlight in WebStorm 😅)